src/scss/constants.scss

```diff
@@ -53,7 +53,7 @@ $small-laptop-width: 980px;
 
 // @debt the following locations are using styled-components, so can't use our helper function yet
 //   pages/Admin/Room/Modal/RoomModal.styles.ts (z-index: 101)
-//   components/molecules/UserProfilePicture/UserProfilePicture.styles.ts (z-index: 10)
+//   components/molecules/UserProfilePicture/UserProfilePicture.styles.ts (z-index: 10, 12)
```

Comment:
What is the change here for? It seems if anything the `10` should be replaced with `12`, based on what I can see in the changes in `src/components/molecules/UserProfilePicture/UserProfilePicture.styles.ts`?